dnYara/Rule.cs

```diff
@@ -10,6 +11,8 @@ namespace dnYara
     /// </summary>
     public sealed class Rule
     {
+        private Nullable<YR_RULE> _struct;
```

Comment:
Same here, why keeping this struct in memory while it's only used over a very short period of time ?

Reply:
Removed in latest.